src/components/views/rooms/RoomTile.js

```diff
@@ -56,7 +56,12 @@ module.exports = createReactClass({
     },
 
     getInitialState: function() {
+         // eslint-disable-next-line camelcase
+        const joinRules = this.props.room.currentState.getStateEvents("m.room.join_rules", "");
+        const joinRule = joinRules && joinRules.getContent() && joinRules.getContent().join_rule;
```

Comment:
just realised this can get simplified further

```suggestion
        const joinRule = joinRules && joinRules.getContent().join_rule;
```

Reply:
`getContent()` always returns an object